library/psa_crypto_slot_management.c

```diff
@@ -345,7 +362,14 @@ psa_status_t psa_validate_key_location( psa_key_lifetime_t lifetime,
 #if defined(MBEDTLS_PSA_CRYPTO_SE_C)
         psa_se_drv_table_entry_t *driver = psa_get_se_driver_entry( lifetime );
         if( driver == NULL )
+        {
+#if defined(MBEDTLS_PSA_CRYPTO_DRIVERS)
```

Comment:
Please re-organize to have only one `#if defined(MBEDTLS_PSA_CRYPTO_DRIVERS)`.

Reply:
Done.